web/static.go

```diff
@@ -73,7 +76,28 @@ func root(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Cache-Control", "no-cache, max-age=31556926, public")
 
 	staticDir, _ := fileutils.FindDir(model.ClientDir)
-	http.ServeFile(w, r, filepath.Join(staticDir, "root.html"))
+	contents, err := os.ReadFile(filepath.Join(staticDir, "root.html"))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
+		return
+	}
+
+	titleTemplate := "<title>%s</title>"
+	strToBeChanged := fmt.Sprintf(titleTemplate, model.TeamSettingsDefaultSiteName)
+	strToBeInserted := strToBeChanged
```

Comment:
These variable names are a bit confusing. Maybe `originalHTML` and `modifiedHTML`?